### PR TITLE
Add deprecation warning when draw.aaline is called with blend=False

### PR DESCRIPTION
**Pull request by zoldalma999**
#592

### CODE REVIEW

**Comment by MightyJosip** [comment]
Closing this just to restart appveyor

File: src_c/draw.c [hunk 1]
```diff
@@ -115,6 +115,13 @@ aaline(PyObject *self, PyObject *arg, PyObject *kwargs)
         return NULL; /* Exception already set. */
     }
 
+    if (!blend) {
+        PyErr_WarnEx(PyExc_DeprecationWarning,
```
Comment (nthykier) [inline src_c/draw.c:119]: Missing check of the return value from `PyErr_WarnEx`  If it returns `-1` then an exception was thrown ( see https://docs.python.org/3/c-api/exceptions.html#issuing-warnings), which in turn requires an early return from this function.

File: src_c/draw.c [hunk 2]
```diff
@@ -264,6 +271,13 @@ aalines(PyObject *self, PyObject *arg, PyObject *kwargs)
                             surf->format->BytesPerPixel);
     }
 
+    if (!blend) {
+        PyErr_WarnEx(PyExc_DeprecationWarning,
```
Comment (nthykier) [inline src_c/draw.c:275]: Missing check of the return value from `PyErr_WarnEx`  If it returns `-1` then an exception was thrown ( see https://docs.python.org/3/c-api/exceptions.html#issuing-warnings), which in turn requires an early return from this function.

**Review by nthykier — CHANGES_REQUESTED** [review]
Thanks for looking at this.  I have added some remarks about the code.

**Pull request closed without merging**